app/services/settings/settings.ts

```diff
@@ -484,9 +500,23 @@ export class SettingsService extends StatefulService<ISettingsServiceState> {
     ];
   }
 
-  setSettings(categoryName: string, settingsData: ISettingsSubCategory[]) {
+  /**
+   * Set settings in obs.
+   * @remark The forceApplyCategory parameter is currently only used for refreshing
+   * video settings before starting recording. This is because the video settings is using the v2 api
+   * and the output settings are currently using the v1 api. This will no longer be needed when
+   * output settings are migrated to the new api.
+   * @param categoryName - name of property
+   * @param settingsData - data to set
+   * @param forceApplyCategory - name of property to force apply settings.
+   */
+  setSettings(
+    categoryName: string,
+    settingsData: ISettingsSubCategory[],
+    forceApplyCategory?: string,
+  ) {
     if (categoryName === 'Audio') this.setAudioSettings([settingsData.pop()]);
-    if (categoryName === 'Video') return;
+    if (categoryName === 'Video' && forceApplyCategory && forceApplyCategory !== 'Video') return;
```

Comment:
This parameter exists because we need to guarantee that the latest video and output settings are in the store when recording. Currently, the only time that a value is passed in is in the `refreshVideoSettings` function that is called right before starting recording. We need to force the store to load the video setting but only in this instance.

Reply:
oh i see, i didn't catch the previous context of always skipping setting video settings before, this can be disregarded